yaml/buildings.yaml

```diff
@@ -18,6 +18,12 @@ globals:
       roof_orientation: {col: "roof:orientation"}
       addr_housenumber: {col: "addr:housenumber"}
       addr_street: {col: "addr:street"}
+  - &building_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail(building)
+  - &building_part_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail("building:part")
```

Comment:
I expect our kind details to more closely match what's in our existing kind list for landuse, and maybe a few POI values. This also lines up with what's on the OSM wiki page: http://wiki.openstreetmap.org/wiki/Key:building.

Some of these are sounding more like `roof:shape` which there's regular (every month it seems) interest in adding to tiles though there is currently no Tangram support for it. http://wiki.openstreetmap.org/wiki/Simple_3D_buildings

`proposed` is interesting, we should do what we do for transit stations for that one.

Should building parts have an additional property to store some of these other random values?